src/include/OpenImageIO/imagebufalgo.h

```diff
@@ -762,58 +787,77 @@ bool OIIO_API fit (ImageBuf &dst, const ImageBuf &src, Filter2D *filter,
-/// image.
+/// be copied from the warped `src`; the default is to alter all the pixels in
+/// `dst`. If `dst` is uninitialized, it will be sized to be an ImageBuf large
+/// enough to hold the warped image if recompute_roi is true, or will have the
+/// same ROI as src if recompute_roi is false. It is an error to pass both an
+/// uninitialized `dst` and an undefined `roi`.
+///
+/// @param dst
+///         The output ImageBuf. If not already initialized, it will be sized
+///         based on `roi` (which itself will default to the size of `src`,
+///         if not specified).
+/// @param src
+///         The source ImageBuf.
+/// @param M
+///         A 3x3 matrix describing the desired spatial transformation
+///         of destination pixel coordinates to source pixel coordinates.
+/// @param roi
+///         The region of `dst` that will receive transformed pixels. If
+///         not specified, it will be all the pixels of `dst`.
+/// @param options
+///         Optional ParamValue's that may control the filtering and
+///         reconstruction.
+///
+/// The `options` list contains optional ParamValue's that may control the
+/// filtering and reconstruction.  The following options are recognized:
+///
+///   - "filtername" : string (default: "")
+///
+///     The type of reconstruction filter used to weight the `src` pixels
+///     falling underneath it for each `dst` pixel.  If the value is the
+///     empty string or not supplied, a reasonable high-quality filter will
+///     be chosen automatically.
+///
+///   - "filterwidth" : float (default: 0)
+///
+///     The width of the reconstruction filter, expressed in pixel units of
+///     the `dst` image. If 0 or not supplied, the default width of the
+///     named filter will be used.
+///
+///   - "wrap" : string (default: "black")
+///
+///     The wrap mode controlling the value of pixel lookups that need to
+///     occur beyond the boundary of the `src` image. (Could be one of:
+///     black, clamp, periodic, mirror.)
+///
+///   - "edgeclamp" : int (default: 0)
+///
+///     If nonzero, will enable special edge clamp behavior to reduce
+///     artifacts at the image edges (experimental).
+///
+///   - "recompute_roi" : int (default: 0)
+///
+///     If nonzero and `dst` is not yet initialized, the result image
+///     will be sized to be large enough to hold the warped image. If
+///     zero (the default), the `dst` image will have the same ROI as
+///     `src`. If the `dst` image already is initialized, its size will
+///     not be chaned and this option will be ignored.
+///
+///   - "filterptr" : pointer to a Filter2D (default: nullptr)
+///
+///     Advanced use: It is also possible to pass a custom reconstruction
+///     filter as a `Filter2D*`, overriding any filtername and filterwidth
+///     that may also be passed. It needs to be be passed as
+///     `ParamValue("filterptr", OIIO::TypePointer), 1, &raw_filter_ptr)`.
```

Comment:
give simpler way for single item? `make_pv`?

Reply:
yes, done and pushed